site/content/overview/resource-validation.md

```diff
@@ -74,53 +27,44 @@ kubectl apply -f coffee-route.yaml
-> If this happens, Step 4 will reject any invalid values (from NGINX perspective).
+{{< note >}}While unlikely, bypassing this validation step is possible if the Gateway API CRDs are modified to remove the validation. If this happens, Step 4 will reject any invalid values (from NGINX perspective).{{< /note >}}
 
-### Step 2 - For Kubernetes 1.25+ - CEL Validation by Kubernetes API Server
+## Step 2 - CEL or Webhook validation by Kubernetes
 
-The Kubernetes API server validates Gateway API resources using CEL validation embedded in the Gateway API CRDs.
-It validates Gateway API resources using advanced rules unavailable in the OpenAPI schema validation.
-For example, if you create a Gateway resource with a TCP listener that configures a hostname, the CEL validation will
-reject it with the following error:
+- **Kubernetes 1.25 and later - CEL validation by Kubernetes API Server**
 
+   The Kubernetes API server validates Gateway API resources using CEL validation embedded in the Gateway API CRDs. It validates Gateway API resources using advanced rules unavailable in the OpenAPI schema validation. For example, if you create a Gateway resource with a TCP listener that configures a hostname, the CEL validation will reject it with the following error:
 
-```shell
-kubectl apply -f some-gateway.yaml
-```
+   ```shell
+   kubectl apply -f some-gateway.yaml
+   ```
 
-```text
-The Gateway "some-gateway" is invalid: spec.listeners: Invalid value: "array": hostname must not be specified for protocols ['TCP', 'UDP']
-```
+   ```text
+   The Gateway "some-gateway" is invalid: spec.listeners: Invalid value: "array": hostname must not be specified for protocols ['TCP', 'UDP']
+   ```
 
-More information on CEL in Kubernetes can be found [here](https://kubernetes.io/docs/reference/using-api/cel/).
+   More information on CEL in Kubernetes can be found [here](https://kubernetes.io/docs/reference/using-api/cel/).
 
-### Step 2 - For Kubernetes 1.23 and 1.24 - Webhook Validation by Gateway API Webhook
 
-The Gateway API comes with a validating webhook which is enabled by default in the Gateway API installation manifests.
-It validates Gateway API resources using advanced rules unavailable in the OpenAPI schema validation. For example, if
-you create a Gateway resource with a TCP listener that configures a hostname, the webhook will reject it with the
-following error:
+- **Kubernetes 1.23 and 1.24 - Webhook validation by Gateway API Webhook**
 
-```shell
-kubectl apply -f some-gateway.yaml
-```
+   The validating webhook must be installed for these Kubernetes versions {{link to installation doc}}. It validates Gateway API resources using advanced rules unavailable in the OpenAPI schema validation. For example, if you create a Gateway resource with a TCP listener that configures a hostname, the webhook will reject it with the following error:
```

Comment:
```suggestion
The validating webhook must be [installed for these Kubernetes versions]({{<relref "installation/installing-ngf/helm/#installing-the-gateway-api-resources">}}. It validates Gateway API resources using advanced rules unavailable in the OpenAPI schema validation. For example, if you create a Gateway resource with a TCP listener that configures a hostname, the webhook will reject it with the following error:
```

Reply:
Why not use the doc from the includes section: https://github.com/nginxinc/nginx-gateway-fabric/blob/main/site/content/includes/installation/install-gateway-api-resources.md?

Reply:
Should this link be specifically to the Helm guide, to the manifests guide, either, or both?

Reply:
```suggestion
   The validating webhook must be installed [with Helm]({{<relref "installation/installing-ngf/helm/#installing-the-gateway-api-resources">}} or [manifests]({{<relref "installation/installing-ngf/manifests/#installing-the-gateway-api-resources">}} for these Kubernetes versions. It validates Gateway API resources using advanced rules unavailable in the OpenAPI schema validation. For example, if you create a Gateway resource with a TCP listener that configures a hostname, the webhook will reject it with the following error:
```

This is my suggestion to provide both options. 
@ADubhlaoich your suggestion is missing 3 spaces in the beginning for the indentation.

Reply:
The instructions for installing the gateway api resources are exactly the same in the helm and manifests docs. I'm not sure why we need both here.

Reply:
I will add 3 spaces to Alan's suggestion and apply that

Reply:
```suggestion
   The validating webhook must be [installed for these Kubernetes versions]({{<relref "installation/installing-ngf/helm/#installing-the-gateway-api-resources">}}. It validates Gateway API resources using advanced rules unavailable in the OpenAPI schema validation. For example, if you create a Gateway resource with a TCP listener that configures a hostname, the webhook will reject it with the following error:
```